src/MSBuild.Abstractions/MSBuildWorkspace.cs

```diff
@@ -104,12 +104,13 @@ private BaselineProject CreateSdkBaselineProject(string projectFilePath, IProjec
                     rootElement.Sdk = DesktopFacts.WinSDKAttribute;
                     break;
                 default:
-                    throw new NotSupportedException("This project has custom imports in a manner that's not supported.");
+                    throw new NotSupportedException($"This project has custom imports in a manner that's not supported. {projectFilePath}");
```

Comment:
can you put single quotes around these paths? It can be hard to distinguish where the path starts in some cases without this.

Reply:
@jmarolf I've added single quotes around paths.